### PR TITLE
Task 11 inject posrtgres into project

### PR DESCRIPTION
**Pull request by MrToady**

### CODE REVIEW

File: src/main/java/com/epam/wl/db/JdbcConnector.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                }
+                Properties properties = new Properties();
+                try (InputStream resourceAsStream = JdbcConnector.class.getResourceAsStream("/jdbc.properties")) {
+                    properties.load(resourceAsStream);
+                    String url = properties.getProperty("url");
+                    String userName = properties.getProperty("userName");
+                    String password = properties.getProperty("password");
+                    instance = new JdbcConnector();
+                    instance.dataSource = new DriverManagerDataSource(url, userName, password);
+                } catch (IOException e) {
```
Comment (TimWinnicheck) [inline src/main/java/com/epam/wl/db/JdbcConnector.java]: most of those variables can be final

File: src/main/java/com/epam/wl/services/UserOrderService.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabase;
+
+import javax.sql.DataSource;
+import java.sql.SQLException;
+import java.util.*;
+
+public class UserOrderService {
+
+    private static UserOrderService instance;
+//    private static DataSource dataSource = MockDBHelper.getEmbeddedDatabase();
```
Comment (TimWinnicheck) [inline src/main/java/com/epam/wl/services/UserOrderService.java]: commented code shouldn't exist - you can always use git to find something you changed

File: src/main/java/com/epam/wl/services/TestUserOrderService.java [hunk 3]
```diff
@@ -28,4 +27,15 @@
...
         return list;
     }
+
+    public static List<Integer> getFreeBookInstancesForThisBook(int bookID) {
+        Random random = new Random();
+        Set<Integer> set = new TreeSet<>();
+        int bound = random.nextInt(15);
+        for (int i = 0; i < bound; i++) {
+            set.add(random.nextInt(27));
+        }
```
Comment (TimWinnicheck) [inline src/main/java/com/epam/wl/services/TestUserOrderService.java]: what is this insanity

File: src/main/java/com/epam/wl/servlets/NewBookOrderServlet.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+public class NewBookOrderServlet extends HttpServlet {
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+
+    }
+
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        System.out.println(request.getRequestDispatcher("bookOption"));
+        System.out.println(request.getRequestDispatcher("exemlarNum"));
+//        request.setAttribute("userorders", TestUserOrderService.getOrders());
+//        request.getRequestDispatcher("userorders.jsp").forward(request, response);
```
Comment (TimWinnicheck) [inline src/main/java/com/epam/wl/servlets/NewBookOrderServlet.java]: see previous comment

**Comment by TimWinnicheck** [comment]
One repetitive issue - the lack of finals. While this might not be a major issue in the training project, it's pretty important for the optimization for them to exist. Then again, in pretty much every production project they just use checkstyle static analyzers, so this might be kinda irrelevant.

**Comment by ginsenghillock** [comment]
Resolve merge conflicts

**Comment by ginsenghillock** [comment]
Reviewed

**Pull request merged**